client/components/marketing-survey/cancel-jetpack-form/jetpack-benefits-step.tsx

```diff
@@ -0,0 +1,159 @@
+		const timeRemaining = getTimeRemainingForSubscription( purchase );
+
+		if ( timeRemaining.months() >= 1 ) {
+			const timeRemainingNumber = timeRemaining.months();
+			const unitOfTime = translate( 'month', 'months', { count: timeRemainingNumber } );
+
+			return { timeRemainingNumber, unitOfTime };
+		} else if ( timeRemaining.weeks() >= 1 ) {
+			const timeRemainingNumber = timeRemaining.weeks();
+			const unitOfTime = translate( 'week', 'weeks', { count: timeRemainingNumber } );
+
+			return { timeRemainingNumber, unitOfTime };
+		}
+
+		const timeRemainingNumber = timeRemaining.days();
+		const unitOfTime = translate( 'day', 'days', { count: timeRemainingNumber } );
+
+		return { timeRemainingNumber, unitOfTime };
+	};
+
+	const renderTimeRemainingString = ( product: ResponseCartProduct | null, purchase: Purchase ) => {
+		// returns early if there's no product or accounting for the edge case that the plan expires today (or somehow already expired)
+		// in this case, do not show the time remaining for the plan
+		const timeRemaining = getTimeRemainingForSubscription( purchase );
+		if ( null === product || timeRemaining.days() <= 1 ) {
+			return null;
+		}
+
+		// if this product/ plan is partner managed, it won't really "expire" from the user's perspective
+		if ( isPartnerPurchase( purchase ) || ! purchase.expiryDate ) {
+			return (
+				<React.Fragment>
+					{ translate(
+						'Your {{strong}} %(productName)s {{/strong}} subscription is still active. {{br/}}',
+						{
+							args: { productName: product.product_name },
+							components: {
+								strong: <strong />,
+								br: <br />,
+							},
+						}
+					) }
+				</React.Fragment>
+			);
+		}
+
+		const translatedPeriod = getTimeRemainingTranslatedPeriod( purchase );
+
+		// show how much time is left on the plan
+		return (
+			<React.Fragment>
+				{ translate(
+					'You still have {{strong}}%(timeRemaining)d %(unitOfTime)s{{/strong}} left on your {{strong}}%(productName)s{{/strong}} subscription. {{br/}}',
+					{
+						args: {
+							timeRemaining: translatedPeriod.timeRemainingNumber,
+							unitOfTime: translatedPeriod.unitOfTime,
+							productName: product.product_name,
+						},
+						components: {
+							strong: <strong />,
+							br: <br />,
+						},
+						comment:
+							"'unitOfTime' is either one of 'day', 'week', 'month', or their plural form. 'timeRemaining' is a number representing the time left that will be used with the 'unitOfTime'.",
+					}
+				) }
+			</React.Fragment>
+		);
+	};
+
+	const getCancelConsequenceByProduct = ( productSlug: string ) => {
+		if ( isJetpackScanSlug( productSlug ) ) {
+			return translate(
+				'Once you cancel, your site will no longer have automatic protection from threats.'
+			);
+		} else if ( isJetpackBackupSlug( productSlug ) ) {
+			return translate( 'Once you cancel, you will lose access to your site backups.' );
+		} else if ( isJetpackSearch( productSlug ) ) {
```

Comment:
@IanRamosC  - In doing some testing - I remembered that `isJetpackSearch` only takes a product object and not a slug string. Alternatively, I have used `JETPACK_SEARCH_PRODUCTS.includes()` elsewhere to match against the array of search product slugs

Reply:
Nice catch! Fixed the usage of that here.